Core/src/com/infiniteautomation/mango/spring/service/DataPointService.java

```diff
@@ -204,7 +204,7 @@ public DataPointVO insert(DataPointVO vo)
 
         if (vo.isEnabled()) {
             // the data point cannot have detectors if it was just inserted, don't query for detectors
-            getRuntimeManager().startDataPoint(new DataPointWithEventDetectors(vo, detectors));
+            getRuntimeManager().startDataPoint(new DataPointWithEventDetectors(vo, detectors), Collections.emptyList());
```

Comment:
I feel like we already did something like this. Was it in in a slightly different spot? Logically this seems correct.

Reply:
Actually Pier, after looking at this some more, it seems like maybe we should insert an empty list/entry into the PointValueCache as well, even if the point is not enabled.

Reply:
so basically remove the if check?

Reply:
@pierpuccini what did we decide about making the same change in the `update` method too. 

@pierpuccini I think what jared is saying would be do to an if else like this:

```
if (vo.isEnabled()) {
             // the data point cannot have detectors if it was just inserted, don't query for detectors
             getRuntimeManager().startDataPoint(new DataPointWithEventDetectors(vo, detectors), Collections.emptyList());
}else {
    this.pointValueCache.updateCache(vo, Collections.emptyList());
}
```

Reply:
@pierpuccini I get it now, with Jared's suggested change we don't want to mess with the update method because we can assume that even if the point was created as disabled it will have an empty cache to start with.

Reply:
Thanks guys for the suggestions, I'll get on them :)